mbl-core/tests/partitions/test_partitions.py

```diff
@@ -126,5 +127,22 @@ def get_actual_part_table():
     return part_table
 
 
+def get_var_for_actual_part(var_name, part_name, default=None):
+    """
+    Get the value of a partition config variable for a partition.
+
+    Args:
+    * var_name (str): name of partition property to get.
+    * part_name (str): name of partition.
+
+    Returns (str): the contents of the config file for the given variable name
+    and partition, or empty string if the config file doesn't exist.
+    """
+    path = part_name / var_name
+    if not path.is_file():
+        return ""
```

Comment:
Can we just let `path.read_text()` throw an exception if the file doesn't exist?

Reply:
Changed to use exceptions and to return a string of "0" in the failure case.

Reply:
Why return "0" in the failure case rather than propogate the exception?

Reply:
Changed to let the exception cause the test to fail.